src/etc/netplan/99-netcheck.yaml

```diff
@@ -0,0 +1,5 @@
+# Set NetworkManager as the default renderer for all adapters.
```

Comment:
"renderer"? Is that the right word? Could be, but I wouldn't expect that to apply to connections.

Reply:
This is the word that netplan uses. I thought it was strange, too.